dockerfiles/build_manylinux_x86_64.Dockerfile

```diff
@@ -94,3 +94,19 @@ RUN which gcc && gcc --version && \
 # We use the wildcard option to disable the checks. This was added
 # in git 2.35.3
 RUN git config --global --add safe.directory '*'
+
+######## vcpkg ########
+WORKDIR /opt
+ENV VCPKG_HASH="3f5ad7be7693ce6ac5599ddb7cc24f260b9d44f9"
+COPY install_vcpkg.sh ./
+RUN yum install -y zip unzip
+RUN ./install_vcpkg.sh "${VCPKG_HASH}"
+
+######## OpenMPI ########
+RUN /opt/vcpkg/vcpkg install openmpi:x64-linux
+ENV PATH="/opt/vcpkg/installed/x64-linux/tools/openmpi/bin:${PATH}"
+ENV PKG_CONFIG_PATH="/opt/vcpkg/installed/x64-linux/lib/pkgconfig:${PKG_CONFIG_PATH:-}"
+ENV LD_LIBRARY_PATH="/opt/vcpkg/installed/x64-linux/lib:${LD_LIBRARY_PATH}"
+
+RUN which mpicc && mpirun && \
+    mpirun --version || true
```

Comment:
Please add a comment here, ideally with a link to an issue, explaining that we eventually want to vendor this dependency and build it alongside the other projects instead of including it in the build dockerfile.

We should then be able to remove this block of code once that is completed.